src/LuaInterface.cpp

```diff
@@ -592,20 +593,20 @@ void LuaInterface::renameVar(TVar* var)
     } else {
         newName.append(R"([")" + vars.back()->getNewName() + R"("])");
     }
-    QString addString = QString(newName + " = " + oldName);
-    luaL_loadstring(L, addString.toLatin1().data());
+    QString variableRenameCode = QString(newName + " = " + oldVariableDeleteCode);
+    luaL_loadstring(L, variableRenameCode.toUtf8().constData());
```

Comment:
I believe we can combining both lines:

    luaL_loadstring(L, QStringLiteral("%1 = %2").arg(newName, oldVariableDeleteCode).toUtf8().constData());